beacon-chain/rpc/prysm/v1alpha1/node/server_test.go

```diff
@@ -89,8 +89,9 @@ func TestNodeServer_GetImplementedServices(t *testing.T) {
 
 	res, err := ns.ListImplementedServices(context.Background(), &emptypb.Empty{})
 	require.NoError(t, err)
-	// We verify the services include the node service  + the 2 registered reflection services.
-	assert.Equal(t, 2, len(res.Services))
+	// Expecting node service and Server reflect. As of grpc, v1.65.0, there are two version of server reflection
+	// Services: [ethereum.eth.v1alpha1.Node grpc.reflection.v1.ServerReflection grpc.reflection.v1alpha.ServerReflection]
```

Comment:
why do we need 2 versions of sever reflection?

Reply:
While updating the bazel dependencies to include the otel packages I was forced to update everything. I tried only pushing the otel changes but the CI didn't like it. So I went through this with Preston and we settled on modifying the comment to be more specific instead.

Reply:
@james-prysm not much we can do about it. They promoted the server reflection API to v1, but they are also serving v1alpha1 for older tooling. It broke our test and the appropriate action was to update the test expectations and add the comment for context.